src/main/java/com/synerzip/projectmanagementapp/model/Employee.java

```diff
@@ -1,27 +1,45 @@
 package com.synerzip.projectmanagementapp.model;
 
+import java.util.List;
+
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
+import javax.persistence.OneToMany;
 import javax.persistence.Table;
+import javax.persistence.Transient;
+
+import org.hibernate.search.annotations.Analyze;
+import org.hibernate.search.annotations.Field;
+import org.hibernate.search.annotations.Index;
+import org.hibernate.search.annotations.Indexed;
+import org.hibernate.search.annotations.Store;
 
-@Entity
-@Table(name = "employee")
+@Entity@Table(name = "employee")@Indexed
 public class Employee {
 
-	@Id
-	@Column(name = "emp_id")
+	@Id@GeneratedValue@Field(index = Index.YES, analyze = Analyze.YES, store = Store.NO)@Column(name = "emp_id")
```

Comment:
why all annotation in same line. 
Please one annotation in one line. 
This comment is valid for controller also.